README.md

```diff
@@ -41,6 +53,42 @@ A valid VRN and continueUrl must be sent in the URL
 |```SEE_OTHER(301)```                     |  ```Redirects to VatRegistrationDate page```       
```

Comment:
Lines 51-53 are missing borders (pipe symbols) at the end. Reformat the table as well once they're added in